<ui version="4.0" >
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Metric Search</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="hostLabel" >
       <property name="text" >
        <string>Host Name Pattern:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="metricLabel" >
       <property name="text" >
        <string>Metric Name Pattern:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="instanceLabel" >
       <property name="text" >
        <string>Instance Name Pattern:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="hostPattern" />
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="metricPattern" />
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="instancePattern" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonAll" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Select &amp;All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultStatus" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>271</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearch" >
       <property name="text" >
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClear" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>C&amp;lear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QListWidget" name="matchList" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>71</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hostPattern</tabstop>
  <tabstop>metricPattern</tabstop>
  <tabstop>instancePattern</tabstop>
  <tabstop>buttonSearch</tabstop>
  <tabstop>matchList</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonClear</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>280</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClear</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>450</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>365</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>450</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hostPattern</sender>
   <signal>textEdited(QString)</signal>
   <receiver>SearchDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>303</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel" >
     <x>229</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metricPattern</sender>
   <signal>textEdited(QString)</signal>
   <receiver>SearchDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>303</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel" >
     <x>229</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instancePattern</sender>
   <signal>textEdited(QString)</signal>
   <receiver>SearchDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>303</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel" >
     <x>229</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAll</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>selectall()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>47</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>matchList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>SearchDialog</receiver>
   <slot>listchanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>230</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
